examples/quickstarts/helidon-quickstart-mp/build.gradle

```diff
@@ -91,3 +80,17 @@ jar {
                )
   }
 }
+
+// This is a work-around for running unit tests.
+// Gradle places resource files under ${buildDir}/resources. In order for
+// beans.xml to get picked up by CDI it must be co-located with the classes.
+// So we move it before running tests.
+// In either case it ends up AOK in the final jar artifact
+task moveBeansXML {
```

Comment:
I'm not keen on using the move option but that's what the Gradle guide recommends

https://docs.gradle.org/current/userguide/working_with_files.html#sec:moving_files_example

Reply:
Yes, I had a similar reaction.